.golangci.yml

```diff
@@ -53,7 +53,6 @@ linters:
   enable:
     - bidichk
     - bodyclose
-    - depguard
```

Comment:
We would need to configure the package to leave it in, but until we have some concrete dependencies that we want to ban (or go to a strict whitelisting system), the plugin isn't helpful.